ports/glutin/window.rs

```diff
@@ -605,15 +607,21 @@ impl WindowMethods for Window {
     }
 
     fn set_page_title(&self, title: Option<String>) {
+        let fallback_title = match self.current_url.borrow_mut().clone() {
```

Comment:
@KiChjang  Thanks for your feedback, changed the code.

I also had to change first line of your suggested code:

`if let Some (ref current_url) = self.current_url.borrow()`
into `if let Some (ref current_url) = self.current_url.borrow().as_ref()` to make it work, made me wonder if there was something I missed or is this ok?